### PR TITLE
Directly output yaml file instead of ost.txt

### PR DESCRIPTION
**Pull request by suryaambrose**

### CODE REVIEW

**Comment by awesomebytes** [comment]
There may be people depending on `ost.txt` as it is, so maybe outputting both would be the best. :)

**Comment by vrabaud** [comment]
yes please, output both types of files. When writing a .txt, display a warning saying it will be removed in Kinetic Kame

**Comment by suryaambrose** [comment]
Finally done (sorry for the delay ^^). 
Now both files are output. 
For mono camera, ost.txt (as it was before) and ost.yaml. 
For stereo camera, ost.txt (as it was before, with both camera in one file) and left.yaml and right.yaml.

File: camera_calibration/src/camera_calibration/calibrator.py [hunk 1]
```diff
@@ -430,9 +430,8 @@ def lrreport(self, d, k, r, p):
         print("R = ", numpy.ravel(r).tolist())
         print("P = ", numpy.ravel(p).tolist())
 
-    # TODO Get rid of OST format, show output as YAML instead
     def lrost(self, name, d, k, r, p):
-        calmessage = (
+	calmessage_ost = (
```
Comment (vrabaud) [inline camera_calibration/src/camera_calibration/calibrator.py]: indentation is wrong

File: camera_calibration/src/camera_calibration/calibrator.py [hunk 2]
```diff
@@ -764,8 +785,9 @@ def taradd(name, buf):
         ims = [("left-%04d.png" % i, im) for i,(_, im) in enumerate(self.db)]
         for (name, im) in ims:
             taradd(name, cv2.imencode(".png", im)[1].tostring())
-
-        taradd('ost.txt', self.ost())
+	yaml_file, ost_file = self.ost()
```
Comment (vrabaud) [inline camera_calibration/src/camera_calibration/calibrator.py]: indentation is wrong too: please use `sudo pip install pep8` and use that executable on your file

File: camera_calibration/src/camera_calibration/calibrator.py [hunk 3]
```diff
@@ -1070,8 +1092,11 @@ def taradd(name, buf):
 
         for (name, im) in ims:
             taradd(name, cv2.imencode(".png", im)[1].tostring())
-
-        taradd('ost.txt', self.ost())
+	l_yaml, l_ost = self.ost("/left", self.l)
+	r_yaml, r_ost = self.ost("/right", self.r)
```
Comment (vrabaud) [inline camera_calibration/src/camera_calibration/calibrator.py]: do not use tabs in Python, use spaces, cf http://wiki.ros.org/PyStyleGuide#Coding_Style

**Comment by awesomebytes** [comment]
Aside of the comments of Vincent, great!

**Comment by suryaambrose** [comment]
Right vrabaud, here it is. 
Thanks awesomebytes ;)

**Comment by vrabaud** [comment]
please `git grep ost` and fix all the occurrences: especially the tests (make sure they pass).
I am not crazy about breaking the API even though people usually only use the calibrator.

**Comment by vrabaud** [comment]
actually, I don't want this API breakage: instead of overloading `ost` and `lrost`, please create the functions:`yaml` and `lryaml`

**Comment by vrabaud** [comment]
thx ! totally late on that one ...

**Pull request merged**